wolfssl/src/error.rs

```diff
@@ -109,7 +109,7 @@ fn wolf_error_string(raw_err: std::ffi::c_ulong) -> String {
             raw_err,
             // note that we are asked for a `char *`, but the
             // following `from_utf8` asks for a Vec<u8>
-            buffer.as_mut_slice().as_mut_ptr() as *mut i8,
+            buffer.as_mut_slice().as_mut_ptr() as *mut std::os::raw::c_char,
```

Comment:
`./target/debug/build/wolfssl-sys-54fbb5ec74fcde7c/out/bindings.rs` has:

```rust
extern "C" {
    pub fn wolfSSL_ERR_error_string(
        errNumber: ::std::os::raw::c_ulong,
        data: *mut ::std::os::raw::c_char,
    ) -> *mut ::std::os::raw::c_char;
}
```

so this is indeed correct.

In C `char` is unsigned on arm compared with Intel which is `signed`, this issue crops up all the time in C too.